website/project/views/node.py

```diff
@@ -718,6 +723,9 @@ def _view_project(node, auth, primary=False):
     anonymous = has_anonymous_link(node, auth)
     widgets, configs, js, css = _render_addon(node)
     redirect_url = node.url + '?view_only=None'
+    contributor_ids = []
+    for contributor in node.contributors:
+        contributor_ids.append(contributor._id)
```

Comment:
No need to change it on this PR, but, for the future, this would be good place for a list comprehension:

```python
contributor_ids = [contributor._id for contributor in node.contributors]
```